src/core/dfn-index.js

```diff
@@ -46,14 +46,14 @@ const CODE_TYPES = new Set([
  * @typedef {{ term: string, type: string, linkFor: string, elem: HTMLAnchorElement }} Entry
  */
 
-export async function run() {
+export function run() {
```

Comment:
Should I keep the async here for consistency? Or, not add async where not needed?

Reply:
I would just remove it.